appveyor.yml

```diff
@@ -235,7 +237,7 @@
 
                 $compatTests = ('Tests\Compat', 'Tests\GrpcCompatClient', 'Tests\GrpcCompatServer')
 
-                msbuild cs\cs.sln /verbosity:minimal "/target:$($compatTests -join ';')" /logger:"C:\Program Files\AppVeyor\BuildAgent\Appveyor.MSBuildLogger.dll"
+                dotnet msbuild cs\cs.sln /verbosity:detailed "/target:$($compatTests -join ';')"
```

Comment:
Is this switch intentional?

Reply:
Everything related to CI is still up in the air, I disabled this since dotnet was not happy with that .dll for some reason. I will track this issue after getting some successful CI runs without this logging dll.

Reply:
It's unsupported for now: https://github.com/appveyor/ci/issues/2212